app/models/ext_management_system.rb

```diff
@@ -26,6 +26,8 @@ def self.supported_types_and_descriptions_hash
   end
 
   belongs_to :provider
+  has_many :child_managers, :class_name => 'ExtManagementSystem', :foreign_key => 'parent_ems_id'
```

Comment:
I called this `child_managers` (which is the opposite of *parent_manager*)
Although I'd like e.g. `dependent_managers` better

Reply:
> Maybe we can define the relation based on parent_ems_id that they all should have?

@moolitayer I dont understand. All child managers are linked via the `parent_ems_id`, so it will catch all, monitoring, network, etc.

Reply:
Just having the child_managers relation will not hurt, since it's just another way to query it(we will still use ext.monitoring_manager and ext.network_manager where needed). But I am not sure if the cascade delete should work the same for all of them?

Reply:
Ahh looks good @durandom I must have missed the `:foreign_key` definitions there.